### PR TITLE
Yep

### PR DESCRIPTION
**Pull request by syncrou**

### CODE REVIEW

**Comment by syncrou** [comment]
@JPrause or anyone - please review.

File: site/_posts/2017-10-06-yep.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: "Last Week in ManageIQ: Yep"
+author: syncrou
+date: 2017-10-10
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Yep, its [syncrou](http://github.com/syncrou) here again for another entry in LWIMIQ.
```
Comment (d-m-u) [inline site/_posts/2017-10-06-yep.md]: You need an apostrophe: ```it's```

**Comment by juliancheal** [comment]
LGTM 👍

**Comment by JPrause** [comment]
Yep! I love it! That was awesome.
I love it that all of you can be creative,...in quirky ways. Ha!

LGTM!

**Comment by JPrause** [comment]
@cybette can you review.

**Comment by cybette** [comment]
Yep, it's a good one!

**Pull request merged**